xpano/gui/panels/about.h

```diff
@@ -1,21 +1,23 @@
 #pragma once
 
 #include <vector>
+#include <future>
 
 #include "utils/text.h"
 
 namespace xpano::gui {
 
 class AboutPane {
  public:
-  explicit AboutPane(std::vector<utils::Text> licenses);
+  explicit AboutPane(std::future<std::vector<utils::Text>> licenses);
   void Draw();
   void Show();
 
  private:
   bool show_ = false;
   int current_license_ = 0;
-  std::vector<utils::Text> licenses_;
+  std::future<std::vector<utils::Text>> licenses_;
+  std::vector<utils::Text> license_text;
```

Comment:
please keep the name licenses_ for the `std::vector<utils::Text>` member, so that there aren't unneccesary code changes.

you can name the new member e.g. licenses_future_